File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, original: str, new: str) -> bool:
        n = set(new)
        if len(original) != len(new):
            return False
        for char in n:
            if new.count(char) != original.count(char):
                return False
        return True
            

File: 0242-valid-anagram/0242-valid-anagram.cs
public class Solution {
    public bool IsAnagram(string s, string t) {
        char[] sArray = s.ToCharArray();
        Array.Sort(sArray);

        char[] tArray = t.ToCharArray();
        Array.Sort(tArray);

        string sortedS = new string(sArray);
        string sortedT = new string(tArray);

        return sortedS == sortedT;
    }
}